x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials.test.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { setMockValues, setMockActions } from '../../../__mocks__/kea.mock';
```

Comment:
Another super small newline nit
```suggestion
 */

import { setMockValues, setMockActions } from '../../../__mocks__/kea.mock';
```

Reply:
d2e21d7407d6477e27304ee1903332e5179d534a